plugins/balloontoolbar/plugin.js

```diff
@@ -760,6 +760,12 @@
 					element.setAttribute( 'draggable', 'false' );
 					this.registerFocusable( element );
 				}, this );
+				// We need to initially set status of richCombo items #1268.
```

Comment:
Can't it be automated by e.g. forcing selection change via [`editor.forceNextSelectionCheck`](https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_editor.html#method-forceNextSelectionCheck)?

Reply:
Doesn't work for inline editors.